firefox_puppeteer/ui/tabbar.py

```diff
@@ -20,7 +20,7 @@ class Tabs(BaseLib):
     @property
     def newtab_button(self):
         """
-        :returns: The new tab button element.
+        :returns: The new tab button element
```

Comment:
Barbara Miller wrote on 02/03/2015 12:36 AM:
>>      def newtab_button(self):
>>          """
>> -        :returns: The new tab button element.
>> +        :returns: The new tab button element
> 
> I'll checkout the master for tabbar.py and resubmit.

The patch has been landed. So please feel free to check and update. Thanks